clang/test/CodeGenSYCL/inline_asm.cpp

```diff
@@ -0,0 +1,34 @@
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -triple spir64-unknown-linux-sycldevice -S -emit-llvm -x c++ %s -o - | FileCheck %s
+
+inline namespace cl {
+  namespace sycl {
+    class kernel {};
+  } // sycl
+} // cl
+
+using namespace cl::sycl;
+
+template <typename name, typename Func>
+__attribute__((sycl_kernel)) void kernel_single_task(Func kernelFunc) {
+  // CHECK: %[[ARRAY_A:[0-9a-z]+]] = alloca [100 x i32], align 4
+  // CHECK-NEXT: %[[NUM:[0-9]+]] = bitcast [100 x i32]* %[[ARRAY_A]] to i8*
```

Comment:
This is only there for the lifetime marker, so no need for htis check line either.

Reply:
Thanks again, Erich!